plugin/action/add_source_name/add_source_name.go

```diff
@@ -0,0 +1,48 @@
+package add_source_name
+
+import (
+	"github.com/ozontech/file.d/fd"
+	"github.com/ozontech/file.d/pipeline"
+	"github.com/ozontech/file.d/plugin"
+)
+
+/*{ introduction
+It adds field containing source name to an event.
+}*/
+
+type Plugin struct {
+	config *Config
+	plugin.NoMetricsPlugin
+}
+
+// ! config-params
+// ^ config-params
+type Config struct {
+	// > @3@4@5@6
+	// >
+	// > The event field to which put the source name. Must be a string.
+	Field string `json:"field" default:"source_name" required:"true"` // *
```

Comment:
What is "source name", if i'm not familiar with internals of file.d? Is it input plugin name? Or what?